packages/assets-controllers/src/NftController.ts

```diff
@@ -382,6 +397,7 @@ export class NftController extends BaseController<NftConfig, NftState> {
         description: null,
         standard: standard || null,
         favorite: false,
+        tokenURI: tokenURI || null,
```

Comment:
Similar as above — does it make sense to say this?

``` suggestion
        tokenURI: tokenURI ?? null,
```

Reply:
Yeah, it can be an empty string, on the mobile code will generate the same result since 'null || ""' will result on '!tokenURI' -> true